FormControlRange/explainer.md

```diff
@@ -0,0 +1,533 @@
+# FormControlRange
+
+## Authors:
+
+- [Andres Regalado Rosas](t-andresre@microsoft.com)
+- [Ana Sollano Kim](https://github.com/anaskim)
+
+## Participate
+
+- [WHATWG tracking issue](https://github.com/whatwg/html/issues/10614)
+- [CSSWG issue tracking initial discussions](https://github.com/w3c/csswg-drafts/issues/10346)
+- [TPAC 2024 WHATWG meeting minutes on initial proposal](https://github.com/whatwg/meta/issues/326#issuecomment-2377500295)
+- [Issue tracker](https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/GetSelectionBoundingClientRect)
+
+## Introduction
+
+The current `Range` interface methods do not support retrieving or creating Range objects that represent the `value` (not the element itself) of `<textarea>` and `<input>` elements. As a result, if web developers want to use the `getBoundingClientRect()` method in a `<textarea>` or `<input>` element to position a popup beneath the user's current caret for delivering contextual autocomplete suggestions or marking syntax errors as users type using the [Custom Highlight API](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Custom_Highlight_API), they must find workarounds. These workarounds often involve cloning these elements and their styles into `<div>`s, which is both difficult to maintain and may impact the web application's performance.
```

Comment:
```suggestion
The current `Range` interface methods do not support retrieving or creating Range objects that represent the `value` (rather than the element itself) of `<textarea>` and `<input>` elements. As a result, if web developers want to use the `getBoundingClientRect()` method in a `<textarea>` or `<input>` element to position a popup beneath the user's current caret for delivering contextual autocomplete suggestions or marking syntax errors as users type using the [Custom Highlight API](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Custom_Highlight_API), they must find workarounds. These workarounds often involve cloning these elements and their styles into `<div>`s, which is both difficult to maintain and may impact the web application's performance.
```